WinUIConvert.md

```diff
@@ -0,0 +1,101 @@
+# Converting To WinUI3 Using Try-Convert
+
+This document details how to use the Try-Convert Tool to convert a C# UWP App to WinUI3 using .NET Native.
+
+## Introduction
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+- WinUI2 is an official library with support for native Windows UI elements for Windows apps. 
+- WinUI3 Update now supports both Windows Desktop and UWP apps. 
+
+Updating a C# UWP App to WinUI3 can be an involved process and requires several changes to the .csproj as well as C# code. Running this tool will automate the process. 
+
+## Description
+This tool assists with the conversion process by first modifying .csproj files to use the new `Microsoft.WinUI` nuget package. It converts nuget packages which are incompatible with the new WinUI3 where possible and removes them otherwise and converts
```

Comment:
missing a '.' or was there more to this?

Reply:
Updated doc